{"name": "Проверка корректной ошибки при некорректной авторизации: {test_data_key}", "status": "passed", "parameters": [{"name": "test_data", "value": "{'email': '', 'password': ''}"}], "start": 1733867477661, "stop": 1733867477905, "uuid": "e336d07e-a37b-4e26-9129-9c91c5050882", "historyId": "fef7277debebb4cdd262ee1977e40572", "testCaseId": "311718a49f0f75c6e906c1fb2ac9a549", "fullName": "tests.test_login user.TestLoginUser#test_login_error_handling", "labels": [{"name": "feature", "value": "Логин пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "Karinas-MacBook-Pro.local"}, {"name": "thread", "value": "8223-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login user"}]}